src/Uno.UI.Wasm/ts/Windows/Devices/Enumeration/Internal/Providers/Midi/MidiDeviceConnectionWatcher.ts

```diff
@@ -0,0 +1,28 @@
+﻿namespace Uno.Devices.Enumeration.Internal.Providers.Midi {
+	export class MidiDeviceConnectionWatcher {
+		private static dispatchStateChanged: (id: string, name: string, isInput: boolean, isConnected: boolean) => number;
+
+		public static startStateChanged() {
+			const midi = Uno.Devices.Midi.Internal.WasmMidiAccess.getMidi();
+			midi.addEventListener("statechanged", MidiDeviceConnectionWatcher.onStateChanged);
+		}
+
+		public static stopStateChanged() {
+			const midi = Uno.Devices.Midi.Internal.WasmMidiAccess.getMidi();
+			midi.removeEventListener("statechanged", MidiDeviceConnectionWatcher.onStateChanged);
+		}
+
+		public static onStateChanged(event: WebMidi.MIDIConnectionEvent) {
+			if (!this.dispatchStateChanged) {
+				this.dispatchStateChanged =
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the parent class name instead of `this` when in a `static` context.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305560)